README.md

```diff
@@ -1,28 +1,169 @@
+[![Bstats server stats](https://img.shields.io/bstats/servers/16576?color=%2300B4DB&style=flat-square&logo=serverless&logoColor=white)](https://bstats.org/plugin/bukkit/VelocityReport/16576)
+[![Bstats player stats](https://img.shields.io/bstats/players/16576?color=%2300B4DB&style=flat-square&logo=odnoklassniki&logoColor=white)](https://bstats.org/plugin/bukkit/VelocityReport/16576)
+[![Modrinth Downloads](https://img.shields.io/modrinth/dt/mqGjZEIE?label=Modrinth&color=blue&style=flat-square&logo=modrinth&logoColor=white)](https://modrinth.com/plugin/velocityreport)
+[![Spigot Downloads](https://img.shields.io/spiget/downloads/105378?label=Spigot&color=blue&style=flat-square&logo=docusign&logoColor=white)](https://www.spigotmc.org/resources/105378)
+[![Polymart Downloads](https://img.shields.io/polymart/downloads/2896?label=Polymart&color=blue&style=flat-square&logo=docusign&logoColor=white)](https://polymart.org/resource/2896)
+[![License](https://img.shields.io/badge/GPL%20V3-blue?label=License&style=flat-square)](https://github.com/Syrent/VelocityReport/blob/master/LICENSE)
+
+### [VelocityReport] aims to be the best report plugin out there.
 
-# Links
-The precompiled JAR can be downloaded for free from:
+### The plugin supports [Velocity]/[Spigot]/[Paper]/[Purpur].
 
-* GitHub releases: https://github.com/Syrent/VelocityReport/releases
-* Modrinth (Recommended): https://modrinth.com/plugin/velocityreport
-* Spigot: https://www.spigotmc.org/resources/velocityreport-1-8-1-19-2-mysql-sqlite.105378/
-* Polymart: https://polymart.org/resource/velocityreport-1-8-1-19-2.2896
-* Paper's forum: https://forums.papermc.io/threads/velocityreport-report-system-for-velocity-servers.515/
+<table>
+<thead>
+<tr>
+<th width="2000" colspan="2">
+</th>
+<h2>🧭 Links</h2>
+</tr>
+</thead>
+<tbody>
+<tr>
+  <td width="80" align="center" valign="top">
+    <br>
+    <a href="https://github.com/Syrent/VelocityReport/wiki"><img src="./blob/images/icon/bookmark.svg"></a>
+  </td>
+  <td valign="top">
+    <h3>Wiki</h3>
+    <p>
+      Don't know how to use it? <a href="https://github.com/Syrent/VelocityReport/wiki">Click here to start getting started</a>!
+      <br>
+      <a href="https://github.com/Syrent/VelocityReport/wiki/Why-VelocityReport">And why Velocity Reoprt</a>?
+    </p>
+  </td>
+</tr>
+  <td width="80" align="center" valign="top">
+    <br>
+    <a href="https://discord.gg/VZk2XU3kFg"><img src="./blob/images/icon/support.svg"></a>
+  </td>
+  <td>
+    <h3>Support</h3>
+    <p>
+      Join our <a href="https://discord.gg/VZk2XU3kFg">discord community</a> for more help.
+    </p>
+  </td>
+</tr>
+<tr>
+  <td width="80" align="center" valign="top">
+    <br>
+    <a href="https://github.com/Syrent/VelocityReport/issues"><img src="./blob/images/icon/bug.svg"></a>
```

Comment:
Missing image

Reply:
When I edit locally with vscode it displays correctly, maybe it's a path issue that's preventing it from displaying correctly